package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "hackathon-starter",
-  "version": "4.2.1",
+  "version": "4.2.2",
```

Comment:
Could you revert this back to 4.2.1? I generally bump it manually and note it in the README after a few changes have been accumulated. I will bump the package to 4.3 after this PR since there have been some bigger changes since the 4.2.1.